### PR TITLE
Move readonly from dev to base

### PR DESCRIPTION
**Pull request by lizhuqi**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
Readonly shall be added to base but not dev so moving it to base in this PR.
**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Mark read-only volumes at provision time.
```

### CODE REVIEW

**Comment by lizhuqi** [comment]
Error:
I0106 23:06:42.348] Waiting for driver to start
I0106 23:06:42.349] [/go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/deploy/kubernetes/wait-for-driver.sh]
W0106 23:07:12.329] error: timed out waiting for the condition on deployments/csi-gce-pd-controller
I0106 23:07:12.430] Deleting driver

It seems that driver failed to start

File: deploy/kubernetes/images/stable-master/image.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: imagetag-csi-provisioner
 imageTag:
   name: k8s.gcr.io/sig-storage/csi-provisioner
-  newTag: "v2.2.1"
+  newTag: "v3.0.0"
```
Comment (mattcary) [inline deploy/kubernetes/images/stable-master/image.yaml:7]: ah, thanks, I forgot there was a version update.

**Pull request merged**